src/PostcodeFormatter.php

```diff
@@ -67,14 +75,21 @@ private function getFormatter(string $country) : CountryPostcodeFormatter
             throw new UnknownCountryException('Unknown country: ' . $country);
         }
 
-        $country = strtoupper($country);
-
-        $class = __NAMESPACE__ . '\\Formatter\\' . $country . 'Formatter';
-
-        if (! class_exists($class)) {
+        if (! $this->isSupportedCountry($country)) {
```

Comment:
I'd prefer to duplicate `class_exists()` here, rather than duplicating the call to `getFormatterClass()`!